sacred/observers/mongo.py

```diff
@@ -297,8 +318,12 @@ def final_save(self, attempts):
                       file=sys.stderr)
 
         from tempfile import NamedTemporaryFile
+        os.makedirs(self.failure_dir, exist_ok=True)
```

Comment:
Actually I just noticed that the `exist_ok` option is py3 only. You'd have to replace it with a manual check.